proto/sentry_protos/snuba/v1/trace_item.proto

```diff
@@ -0,0 +1,28 @@
+syntax = "proto3";
+
+package sentry_protos.snuba.v1;
+
+import "google/protobuf/timestamp.proto";
+import "opentelemetry/proto/common/v1/common.proto";
+
+enum TraceItemType {
+  TRACE_ITEM_TYPE_UNSPECIFIED = 0;
+  TRACE_ITEM_TYPE_SPAN = 1;
+  TRACE_ITEM_TYPE_ERROR = 2;
+  TRACE_ITEM_TYPE_LOG = 3;
+  TRACE_ITEM_TYPE_UPTIME_CHECK = 4;
+}
+
+message TraceItem {
+  uint64 organization_id = 1;
+  uint64 project_id = 2;
+  string trace_id = 3;
+  uint64 item_id = 4;
```

Comment:
don't we store item_id as a Uint128?

Reply:
We do, we do, I need to change that, I wanted to make something that was generating the protos. I think I'll replace that with a `bytes` type since we don't have `u128`.